flan/src/routes/chat/+page.svelte

```diff
@@ -155,6 +154,27 @@
 		inputFile = null;
 	}
 
+	function onConnClose(): void {
+		console.log('Connection closed');
+		const toastMessage: ToastSettings = {
+			message: 'Session disconnected 😿',
+			background: 'variant-filled-warning',
+			autohide: false
+		};
+		toastStore.trigger(toastMessage);
+	}
+
+	async function handlePaste(event: ClipboardEvent) {
+		const clipboardData = event.clipboardData;
+		const items = clipboardData?.files;
+		if (!items) {
+			return;
+		}
+		for (let item of items) {
```

Comment:
Here you are constantly reassigning inputFile to a newer element to the list, is the intention to  just choose the last object in the list since that's what's happening happening?

Reply:
honestly, this logic is pretty trash so, defs needs a whole re-write -- I just saw that it worked and chucked it in. will update once I find a better way

Reply:
Addressed now